spec/engine/miq_ae_method_service/miq_ae_service_model_base_spec.rb

```diff
@@ -92,4 +98,12 @@ module MiqAeServiceModelSpec
   class TestInteger < ::Integer; end
   class TestApplicationRecord < ::ApplicationRecord; end
   class TestVmOrTemplate < ::VmOrTemplate; end
+  class TestPrivateAttrExpose < ::ApplicationRecord
+    self.table_name = 'generic_objects'
+
+    def properties
+      'private attr properties'
+    end
```

Comment:
I don't think we need to define this method since a getter method is exposed from ActiveRecord.  Just need to define the `private :properties` in the model.